app/views/steps/client/has_nino/edit.html.erb

```diff
@@ -5,16 +5,18 @@
   <div class="govuk-grid-column-two-thirds">
     <%= govuk_error_summary(@form_object) %>
 
+    <h1 class="govuk-heading-xl"><%=t '.heading' %></h1>
+
     <%= step_form @form_object do |f| %>
-      <%= f.govuk_radio_buttons_fieldset :has_nino do %>
-        <%= f.govuk_radio_button :has_nino, YesNoAnswer::YES, link_errors: true do %>
-          <%= f.govuk_text_field :nino, width: 'three-quarters' %>
-        <% end %>
 
-        <%= f.govuk_radio_button :has_nino, YesNoAnswer::NO %>
-      <% end %>
+      <%= f.govuk_text_field :nino, width: 'three-quarters', label: nil %>
```

Comment:
I think for accessibility reasons, the label must be always set, however hidden (`label: { hidden: true }`).
The label can be just "Enter client's national insurance number" (comes from locales), anything representative of what the input is for as it is read aloud on focus by screen readers.

Also minor thing but feels like `width: 'three-quarters'` is too big for NINO? maybe `one-third`?